storage/range.go

```diff
@@ -1035,6 +1069,78 @@ func (r *Range) handleSkippedIntents(args proto.Request, intents []proto.Intent)
 	}
 }
 
+type chainedError struct {
```

Comment:
added a TODO. Convenient to do this when actually doing something with `setCorrupt`.